api/server/router/network/network_routes.go

```diff
@@ -70,6 +71,8 @@ SKIP:
 		// info is available for network specific GET API (equivalent to inspect)
 		if versions.LessThan(httputils.VersionFromContext(ctx), "1.28") {
 			nr = n.buildDetailedNetworkResources(nw, false)
+		} else if netFilters.Contains("dangling") {
+			nr = n.buildDetailedNetworkResources(nw, true)
```

Comment:
Oh, we were discussing this; i.e. can we find a solution where we would still return the same information (i.e., no "detailed" information)? https://github.com/moby/moby/pull/31551#discussion_r162113560